cloudferrylib/os/image/glance_image.py

```diff
@@ -190,10 +190,10 @@ def image_exists(self, image_id):
             except glance_exceptions.HTTPNotFound:
                 return False
 
-    def get_image_list(self):
+    def get_image_list(self, is_filtering=True):
         images = self.glance_client.images.list(filters={"is_public": None})
```

Comment:
Boolean arguments considered harmful:
http://ariya.ofilabs.com/2011/08/hall-of-api-shame-boolean-trap.html
http://silkandspinach.net/2004/07/15/avoid-boolean-parameters/
http://blog.gfader.com/2010/06/avoid-boolean-parameters-in-method.html